### PR TITLE
added GetByExternAccount method

### PR DESCRIPTION
**Pull request by yvz5**

### CODE REVIEW

File: simplic-authentication/Simplic.Authentication/Interface/IAuthenticationService.cs [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
         /// <returns>A user session if the user could be logged in, else an exception will be thrown</returns>
-        Simplic.Session.Session Login(string domain, string userName, string password);
+        Session.Session Login(string domain, string userName, string password);
+
+        /// <summary>
+        /// Authenticate a user and create a user session
+        /// </summary>
+        /// <param name="externAccountName">External account name</param>
+        /// <returns>A user session</returns>
+        Session.Session LoginByExternAccount(string externAccountName);
```
Comment (simplicbe) [inline simplic-authentication/Simplic.Authentication/Interface/IAuthenticationService.cs]: Is the account name enough?

Comment (yvz5) [reply simplic-authentication/Simplic.Authentication/Interface/IAuthenticationService.cs]: account typeid was not used at all. so I removed it

Comment (simplicbe) [reply simplic-authentication/Simplic.Authentication/Interface/IAuthenticationService.cs]: What do you mean with not used? The external account system is made for this. We should take a look at it together. I think for office 365 this was used.

Comment (yvz5) [reply simplic-authentication/Simplic.Authentication/Interface/IAuthenticationService.cs]: check this line here: https://github.com/simplic/simplic-framework/blob/8c6079fdba6c5888451af0ffbbf033a34fca0fb1/src/Simplic.Framework.UI/Core/User/UserManager.cs#L306

**Pull request merged**